tapeagents/llms.py

```diff
@@ -758,7 +761,7 @@ def get_logprobs_chat_complete(self, prompt: Prompt, output: LLMOutput) -> dict[
             "n": 1,  # number of completions to generate
             "stream": False,  # return a single completion and not a stream of lines
         }
-        base_url = self.base_url if isinstance(self.base_url, str) else random.choice(self.base_url)
+        base_url = (self.base_url if isinstance(self.base_url, str) else random.choice(self.base_url)).rstrip("/")
```

Comment:
Side note: looks like we have this line repeated 3 times in a single module. We should take a look at it later, it's definitely something to be rewritten for simplicity.